modules/library/metadata/src/test/java/org/geotools/util/factory/GeoToolsTest.java

```diff
@@ -164,7 +164,7 @@ public void testVersion() {
 
         version = GeoTools.getVersion(Geometry.class);
         assertNotNull(version);
-        assertTrue(version.toString().startsWith("1.17"));
+        //        assertTrue(version.toString().startsWith("1.17"));
```

Comment:
This should be fixed or removed.

Reply:
I kind of want to keep it, we do need to upgrade each time we change the jts library - but I do want to check the version number of a "normal jar" here.

Reply:
you know the case being tested here (extracting version number from jar name) is covered by JAI.class above